scss/_styles.scss

```diff
@@ -2,10 +2,15 @@
   font-family: 'Roboto', sans-serif;
 }
 
+body {
```

Comment:
To add a style to the body aside from the equalizer is not really good practice. Try to set it to the container

Reply:
Done 😄